.github/workflows/build.yml

```diff
@@ -156,7 +156,7 @@ jobs:
           name: plugins.tsv
           path: ${{ env.BUILD_BASE }}/plugins.tsv
           # We don't really care about this artifact, its presence is a flag to the post-build job.
-          retention-days: 1
+          retention-days: ${{ github.ref == 'refs/heads/trunk' && 7 || 1 }}
```

Comment:
Either update the comment here too, or just leave this one at 1 day for everything.

This artifact really only exists for the Post Build workflow run, so it can exit early if no plugins were built instead of having to download the whole jetpack-build artifact to determine that.